theme/default/css/ms2-theme.css

```diff
@@ -494,3 +494,27 @@ div.ol-scale-line-inner{
 	margin-bottom: 10px;
 	padding: 15px;
 }
+
+
+.modal-header button.close {
+    margin-top: -7px;
+    opacity: 1;
+}
+
+.modal-header button.close:hover {
+    opacity: 0.5;
+    color: white;
+}
+
+.modal-header button.close:focus {
+    opacity: 0.5;
+    color: white;
+}
+.modal {
+	overflow-y: auto !important;
```

Comment:
!important is too restrictive. Sometimes we don't want overflow-y for every modal. Can you understand where it is used, and replace the css query in something like `div.modal`, that can be replaced in a easy way?